charts/sapphire/templates/pdb.yaml

```diff
@@ -0,0 +1,21 @@
+{{- if .Values.podDisruptionBudget.enabled }}
+---
+
+{{- if $.Capabilities.APIVersions.Has "policy/v1" }}
+apiVersion: policy/v1
+{{- else }}
+apiVersion: policy/v1beta1
+{{- end }}
```

Comment:
Вот этот вопрос я задал в гугл доке - какой k8s мы хотим поддерживать.

Reply:
https://confluence.2gis.ru/display/Karta/On-Premise+infrastructure+requirements+for+OSS911+Eastern+Province+territory
Тут указано Kubernetes v1.19
Убираем подобного рода условия?

Reply:
Да, наше требование, чтобы был k8s 1.19 или выше. Поэтому можно убрать условие

Reply:
Ошибочка вышла-  policy/v1 API version, available since v1.21
https://kubernetes.io/docs/reference/using-api/deprecation-guide/#poddisruptionbudget-v125

Reply:
Но я удалил то лишнее https://github.com/2gis/on-premise-helm-charts/pull/9/commits/92bedc0092b87a0f7db4f68a7cbf8ca2b83fa1a0